app/src/main/java/com/example/NotificationCenter.java

```diff
@@ -0,0 +1,39 @@
+import android.content.Context;
```

Comment:
missing some imports like the package import, and other imports like the R import.